pki.spec

```diff
@@ -433,6 +432,11 @@ Requires:         resteasy-core >= 3.0.17-1
 Requires:         resteasy-jackson2-provider >= 3.0.17-1
 %endif
 
+%if 0%{?is_java_11}
```

Comment:
We probably can use this instead:
```
%if 0%{?fedora} && 0%{?fedora} >= 33
```

Reply:
But, that assumes we always build with JDK11 on F33. If we ever want to move back to JDK8, we'll still pull in these dependencies anyways... even though we don't need to (and it isn't clear if they'll be used). Seems better to tie it somehow to `min_java_version` (and compare to `1:11` somehow?) in that case. Can we just do `%if %min_java_version > 1:1.8`?

Reply:
If that works sure we can use it. But regardless, the _current_ assumption is that we will build with JDK11 on F33. If that changes in the future, we can always change it again or even remove the clause if it's no longer needed.

Reply:
Ah I forgot, that probably won't work because it is a string comparison, not a number comparison. Ok, taking this suggestion, thanks!